src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -1109,13 +1109,14 @@ def retrieve_token_for_service_principal(self, sp_id, resource, tenant, use_cert
         token_entry = sp_auth.acquire_token(context, resource, sp_id)
         return (token_entry[_TOKEN_ENTRY_TOKEN_TYPE], token_entry[_ACCESS_TOKEN], token_entry)
 
-    def retrieve_secret_of_service_principal(self, sp_id):
+    def retrieve_cred_for_service_principal(self, sp_id):
```

Comment:
Replace `of` with `for` to be in sync with `retrieve_token_for_service_principal`.